Energy Web/energy-web-mainnet-starter/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "energy-web-starter",
+  "version": "0.0.1",
+  "description": "This project can be use as a starting point for developing your new Aurora EVM SubQuery project",
```

Comment:
Missing correct network name

Reply:
`"description": "This project can be use as a starting point for developing your new {{prettyName}} SubQuery project",`